ConductionNL/grafregistratiecomponent
# development

- Use the same calendar
- Subpath ingress
- Take version into account on deploy
- Up
- Added postman collection
- build
- updated postman collection
- Add postman checking
- Uups
- Add OpenZaak componenten
- Auto deploy to dev zuid-drecht
- Bump symfony/http-kernel from 4.4.8 to 4.4.13 in /api
- NginX Fix
- Property configuration added to entity Cemetery
- Fixjes en cgb update
- Let's see if this works
- Update postman
- Update CGB
- Helm fix
- Helm
- Update CGB
- Another CGB update
- CGB update
- Test workflow change
- Let's try it like this
- also push
- fixje?
- fixje!
- another fix attempt
- Another try on tagging without building
- Remove some merge conflict
- remove unnecessary )
- Add condition to tagging
- Check if tags are set correctly
- Is APP_BUILD correct?
- Export fixje
- more fixes
- foutje
- Try with different location
- only tag conduction images
- Turn around tags
- re-enable pushing
- Also push versioned images
- Uppercase
- small fix on ingress
- Meertaligheid van resources
- Fix Gedmo
- revert gedmo fix, upgrade symfony/flex
- Hard fix on Gedmo
- Added graphgq interfaces to the security exemptions
- APP_URL env
- API ui fixes
- .env update
- Symfony 5.1 upgrade
- Helm update
- Apply fixes from StyleCI
- Correct dockerfile names
- Fixes
- update cgb
- fix helm
- update cgb again
- Helm fixje
- Add postgresql dependency when Postgres is enabled
- Update ingress
- Pull latest when on prod
- Comply to new github-workflow rules
- Remove all set-envs
- Gefixte workflow
- Fix for links in api description
- Fixje
- Update installation manual
- Few textual errors fixed
- En nog een heel klein fixje
- Updating SECURITY.yaml
- First draft CONTRIBUTING.md
- typo
- A bit about haven
- Fill authors file
- add software dependencies
- System requirements and do away with travis
- Updates
- Aanvullingen
- Contributing
- Updates
- Update composer.lock
- Local subscriber and bin/console
- Fixes
- Remove old postman collection
- Apply fixes from StyleCI
- auto doc update
